a10_neutron_lbaas/v1/handler_member.py

```diff
@@ -35,14 +36,17 @@ def _create(self, c, context, member):
         server_ip = self.neutron.member_get_ip(context, member,
                                                c.device_cfg['use_float'])
         server_name = self._meta_name(member, server_ip)
-
+        conn_limit = c.device_cfg['conn-limit']
+        if conn_limit > 8000000 or conn_limit < 1:
+            raise a10_ex.ConnLimitOutOfBounds("Conn-limit must be >1 and <8000000")
```

Comment:
Possibly, as this deals with the connection limit to servers, I didn't want to fail quietly and set a default value. I believe that the user should be alerted one way or another that they have specified an out of bounds connection limit. When in invalid limit is set on the WebUI, an error is raised that halts the connection limit from being changed and informs the user right away.